models/notification.schema.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+  _id: String,
```

Comment:
Delete this, I think we should use ObjectId for the _id and mongo will generate it for us without us explicitly putting it in the schema